### PR TITLE
Added tlg0086.tlg021

### PR DESCRIPTION
**Pull request by planatheisa**
Checking Hook.

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +6394 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0086/tlg021/__cts__.xml`                   | New      |
| `./data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml` | New      |
| `./data/tlg0086/tlg021/__cts__.xml`                   | Passing  |
| `./data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3732)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +6394 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0086/tlg021/__cts__.xml`                   | New      |
| `./data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml` | New      |
| `./data/tlg0086/tlg021/__cts__.xml`                   | Passing  |
| `./data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3736)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +6394 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0086/tlg021/__cts__.xml`                   | New      |
| `./data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml` | New      |
| `./data/tlg0086/tlg021/__cts__.xml`                   | Passing  |
| `./data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3745)*

File: data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,955 @@
...
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Aristotelis De animalium motione et De animalium incessu</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Vernerus Guilelmus Jaeger</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg0086">Aristoteles</author>
```
Comment (sonofmun) [inline data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml]: Needs an @xml:lang attribute on the author element.

File: data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,955 @@
...
+                            <imprint>
+                                <publisher>Teubner</publisher>
+                                <pubPlace>Leipzig</pubPlace>
+                                <date>1913</date>
+                            </imprint>
+                            <biblScope unit="volume"/>
```
Comment (sonofmun) [inline data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml]: Since there is no content to this element (biblScope), I would delete it.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +19 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +6394 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0086/tlg021/__cts__.xml`                   | New      |
| `./data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml` | New      |
| `./data/tlg0086/tlg021/__cts__.xml`                   | Passing  |
| `./data/tlg0086/tlg021/tlg0086.tlg021.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3798)*

**Pull request merged**